mailpoet/build.sh

```diff
@@ -74,7 +64,6 @@ rm -rf vendor-prefixed/symfony/dependency-injection/Extension
 
 # Copy release folders.
 echo '[BUILD] Copying release folders'
-cp -Rf lang $plugin_name
```

Comment:
I think we still need to copy this folder because we distribute `.pot` file in it. Seehttps://mailpoet.atlassian.net/browse/MAILPOET-4028

Reply:
Nice catch! I removed commit, which contains this change.